ballerina/tests/secure_client_endpoint_test.bal

```diff
@@ -309,3 +309,38 @@ public function testSecureDeleteFileContent() returns error? {
             msg = "Correct error is not given when trying to get a non-existing file.");
     }
 }
+
+@test:Config {
+    dependsOn: [testSecurePutFileContent]
+}
+public function testSecureFileStreamReuse() returns error? {
+    stream<io:Block, io:Error?> localFileStream = check io:fileReadBlocksAsStream(putFilePath, 5);
+    check (<Client>sftpClientEp)->put("/tempFile1.txt", localFileStream);
```

Comment:
Can't we check whether the `sftpClientEp` is `Client` first. Then we can avoid `<Client>` castings